editoast/src/views/infra/auto_fixes.rs

```diff
@@ -792,4 +818,53 @@ mod test {
             obj_type: ObjectType::BufferStop,
         })));
     }
+
+    #[rstest::rstest]
+    async fn missing_track_extremity_buffer_stop_fix() {
+        // GIVEN
+        let app = create_test_service().await;
+        let empty_infra = empty_infra(db_pool()).await;
+        let empty_infra_id = empty_infra.id();
+
+        let track: RailjsonObject = TrackSection {
+            id: "track_with_no_buffer_stops".into(),
+            length: 1_000.0,
+            ..Default::default()
+        }
+        .into();
+        let req_create = get_create_operation_request(track.clone(), empty_infra_id);
+        assert_eq!(
+            call_service(&app, req_create).await.status(),
+            StatusCode::OK
+        );
+
+        // WHEN
+        let req_fix = TestRequest::get()
+            .uri(format!("/infra/{empty_infra_id}/auto_fixes").as_str())
+            .to_request();
+        let response = call_service(&app, req_fix).await;
```

Comment:
A helper was added, so you can do:
```suggestion
        let response = call_service(&app, auto_fixes_request(empty_infra_id)).await;
```